MeetU/MeetU/Views/Account/Login.cshtml

```diff
@@ -5,7 +5,7 @@
 }
 
 <div class="login-content-wrapper">
-    <div class="login-phone-mode-homepage-wrapper">
+    <div class="login-phone-mode-homepage-wrapper" onclick="scrollLogin(900)">
```

Comment:
usually we don't use onclick or onsubmit. you can use jquery $(element).on() or $(element).click() or, in angular, ng-click=""

similarly, we don't use embedded style and script, neither directly set width="" or height="" in html.